Cargo.toml

```diff
@@ -3,7 +3,6 @@ members = [
   "ic-factory",
   "ic-auction",
   "ic-helpers",
-  "ic-stable-storage",
```

Comment:
intentional?

Reply:
Good thing you asked about this. I've completely forgot about this change. But my question is, is this crate supposed to be included? `cargo check` fails with it, because it has wrong `ic` dependency. Did someone forget something when he did something?